src/components/RouteCard/index.jsx

```diff
@@ -0,0 +1,25 @@
+function RouteCard({journeyInfo}) {
+
+    return (
+        journeyInfo && journeyInfo.length > 0 &&
+        <div className='container grid-cols-2 flex justify-evenly gap-3 border-2 rounded-3xl shadow'>
+            <div className='font-Anton'>
+                <div>
+                    <h2>From: {journeyInfo[0].from}</h2>
+                    <h2>To: {journeyInfo[0].to}</h2>
```

Comment:
If you're only ever using the first element from `journeyInfo`, why pass in an array at all?

Reply:
This is groundwork for a later story. Happy to discuss